docs/cloud-guidance/azure/ukho-azure-tagging.md

```diff
@@ -0,0 +1,28 @@
+# UKHO Azure Tagging Strategy v0.1 DRAFT
+
+## How to use this document
```

Comment:
```suggestion
```

This can go to be honest. Just extra words that need reading that aren't helpful